spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :user do
+    name "MyString"
+    email "MyString"
```

Comment:
wpisuj email w dobrym formacie w factory bo pozniej jak dodasz validacje to wybuchna testy ;)